lib/core/random_image.dart

```diff
@@ -0,0 +1,14 @@
+// ignore_for_file: unused_import
+
+import 'package:flutter/material.dart';
+
+class RandomImage extends StatelessWidget {
+  const RandomImage({super.key, this.height = 100});
+  final imageUrl = "https://picsum.photos/200/300";
```

Comment:
Prefer static const here.

```suggestion
  static const imageUrl = "https://picsum.photos/200/300";
```

Reply:
Done.